server/app/models/fusor/deployment.rb

```diff
@@ -12,8 +12,5 @@
 
 module Fusor
   class Deployment < ActiveRecord::Base
-    serialize :rhev_params, Hash
```

Comment:
We need to define the organization and lifecycle environment relationships in the model.

Something like: 
```
belongs_to :organization
belongs_to :lifecycle_environment, :class_name => "Katello::KTEnvironment"
```